README.rst

```diff
@@ -1,13 +1,16 @@
-Astronomical data analysis development leveraging Jupyter platform
-------------------------------------------------------------------
+Astronomical data analysis leveraging the Jupyter platform
+----------------------------------------------------------
 
 .. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
 
-Jdaviz is a data analysis tool which can be used with Jupyter Notebook
-and as a web application.
+JDAViz is a visual data analysis tool that can be used either within a Jupyter 
+Notebook or as a standalone web application. Note that the package is under 
```

Comment:
```suggestion
Notebook or as a standalone application. Note that the package is under 
```
Since the scope is eventually to have it be something that users don't necessarily *know* is web, I think we can just leave it ambiguous here